src/NServiceBus.AcceptanceTests/Audit/When_audit_is_everriden_in_code.cs

```diff
@@ -0,0 +1,70 @@
+﻿namespace NServiceBus.AcceptanceTests.Audit
+{
+    using System;
+    using NServiceBus.AcceptanceTesting;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_audit_is_everriden_in_code : NServiceBusAcceptanceTest
+    {
+        [Test]
+        public void Should_audit_to_target_queue()
+        {
+            var context = new Context();
+            Scenario.Define(context)
+                .WithEndpoint<UserEndpoint>(b => b.Given(bus => bus.SendLocal(new MessageToBeAudited())))
+                .WithEndpoint<AuditSpy>()
+                .Done(c => c.MessageAudited)
```

Comment:
I've been bitten by this a few times and I can never remember. Will the test blow up if no time limit is set?

If not this will be a false positive since there is no assert